### PR TITLE
Update titles, pipeline images and tracking intro

### PR DESCRIPTION
**Pull request by jansenbob**
* Update the pipeline images to better tell which part you're looking at in the docs
* Update the introduction of tracking to close the gap with other intros
* Update the titles of all introductions to provide a bit more information than **introduction** and make them discernable.

Todo: add a section that explains why we want you to map your application to the taxonomy in the Tracking section.

### CODE REVIEW

File: docs/docs/introduction.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ In order to run Objectiv for local development, we'll help you set up the follow
 * The **Objectiv Collector** and a **PostgreSQL data store** to collect, validate & store event data from the tracker.
 * A **Notebook** with the **Objectiv Bach** modeling library to explore and model your data.  
 
-<img src={useBaseUrl('/img/objectiv-pipeline.svg')} alt="Objectiv Pipeline" class="img-l" />
+<img src={useBaseUrl('/img/objectiv-pipeline.svg')} alt="Objectiv Pipeline" class="img-l" />;
```
Comment (ivarpruijn) [inline docs/docs/introduction.md:35]: the `;` should be removed here

**Review by ivarpruijn — CHANGES_REQUESTED** [review]
Looks great! Just one very minor comment

File: docs/docs/introduction.md (hunk 1 above)
Comment (jansenbob) [reply docs/docs/introduction.md]: done

**Review by ivarpruijn — APPROVED** [review]
Looks good

**Comment by jansenbob** [comment]
minor bug: For contexts that have `ItemContext` as a parent or grandparent, `ItemContext` is not clickable.

So that includes:
InputContext
ActionContext
ButtonContext
LinkContext

The rest LGTM

**Pull request merged**